app/advancedfilters/advancedFilters.js

```diff
@@ -194,5 +199,11 @@ app.controller('AdvancedFiltersCtrl', function($scope, $rootScope, $routeParams,
       $scope.view[type] = !$scope.view[type];
     }
 
+    $scope.toggled = function(open) {
+      if(!open) {
+        $scope.updateFilters();
```

Comment:
This is also refreshing the annotations list but there is not need here. Refreshing here should be avoided.